### DISCUSSION TITLE
Remove the `{{link}}` macro

### DISCUSSION THREAD

**Discussion by MrBrain295** [mdn/mdn-community#187] [Platform] [answered]
The `{{link}}` macro isn't necessary or widely used it needs to be removed.

**Comment by teoli2003** [comment, marked as answer]
Yes, it should be removed. We have 3 left on mdn/content.

I think this is a remnant of the Wikimedia era.

Edit:
@yin1999 removed the last occurrence on MDN: mdn/content#19447
@MrBrain295 marked them as deprecated in mdn/yari#6865
